extension/test/unit/markOrderAsShopgateTest.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+const sinon = require('sinon')
+const chai = require('chai')
+const Logger = require('bunyan')
+const BigCommerce = require('node-bigcommerce')
+
+chai.use(require('chai-subset'))
+chai.use(require('chai-as-promised')).should()
+
+const BigCommerceFactory = require('../../lib/bigcommerce/Factory')
+const markOrderAsShopgate = require('../../lib/markOrderAsShopgate')
+
+describe('markOrderAsShopgate', () => {
+  const sandbox = /** @type sinon */sinon.createSandbox()
+  const context = { config: {} }
+  let apiClientStub
+
+  beforeEach(() => {
+    context.log = sandbox.createStubInstance(Logger)
+    apiClientStub = sandbox.createStubInstance(BigCommerce)
+    sandbox.stub(BigCommerceFactory, 'createV2').returns(apiClientStub)
+  })
+
+  afterEach(() => {
+    sandbox.verifyAndRestore()
+  })
+
+  it('should do something', async () => {
```

Comment:
what should it do :)

Reply:
done

Reply:
oops, that should not have been committed... 😂 
...done